src/redux/data.js

```diff
@@ -973,6 +976,7 @@ const INITIAL_STATE = {
     received: [],
   },
   transactions: [],
+  transactionSignatures: {},
```

Comment:
currently, this is unused - but we may want to use redux instead of localstorage every time

Reply:
yep, added